saleor/plugins/webhook/plugin.py

```diff
@@ -470,6 +470,11 @@ def __run_payment_webhook(
         previous_value,
         **kwargs
     ) -> "GatewayResponse":
+        """Trigger payment webhook event.
+
+        Only one app should have defined the webhook for payment event.
+        If more than one app has, the webhook is sent only for the first one.
```

Comment:
I think we should consider blocking this behavior, or at least give a warning at the point of creation, of course not in the scope of this PR. WDYT?

Reply:
This is current behavior, I just added the docstring to clarify what happened. We can consider smth like that but it should be a separate ticket